### DISCUSSION TITLE
Bark integration for TTS would be awesome

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#1532] [General]
Better even than the commercial Elevenlabs, Bark as an addition to the webui would make TTS sound like talking to a real person with emotions, reactions, and true fun in multiple languages! Below is a demonstration:

https://www.youtube.com/watch?v=rU5Do9yHbwM

**Comment by BarfingLemurs** [comment]
This one extension I tried.
https://github.com/wsippel/bark_tts

(speaker 1), default settings
<video src='https://user-images.githubusercontent.com/128182951/234301657-7c4ec2da-6078-44bb-8928-71b37bd8bc9a.mp4' />


(speaker 5), default settings (faster talker)
<video src='https://user-images.githubusercontent.com/128182951/234306270-09fdf859-a582-4360-a2e7-f80dc3a15d1e.mp4' />

**Reply by sarutobiumon** [reply]
Awesome work, checking it out now.
